src/components/fullscreen-flyout/OverlayHeader.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../styles/variables';
+
+.overlay-header {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 13px 16px;
+    background-color: white;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, .08);
+
+    .oh-text {
+        .oh-primary-title {
+            color: $bdl-gray;
+            font-weight: bold;
+            font-size: 16px;
+        }
+
+        .oh-secondary-title {
+            color: $bdl-gray-62;
+            font-size: 12px;
+        }
+    }
+
+    .oh-buttons {
+        .close-btn {
+            width: 32px;
+            height: 32px;
+            padding: unset;
+            background-color: $bdl-gray-10;
+            border: unset;
+        }
+    }
+
+    //> .aria-menu {
```

Comment:
Is this intended to be removed since they are commented out?

Reply:
ah. this one fell through the cracks. It was intended to be removed. Thanks for catching that